specification/vmware/resource-manager/Microsoft.AVS/preview/2020-07-17-preview/vmware.json

```diff
@@ -0,0 +1,4002 @@
+        "revision": {
+          "description": "NSX revision number.",
+          "type": "integer",
+          "format": "int64"
+        }
+      }
+    },
+    "WorkloadNetworkPortMirroring": {
+      "description": "NSX Port Mirroring",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/WorkloadNetworkPortMirroringProperties",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "WorkloadNetworkPortMirroringList": {
+      "description": "A list of NSX Port Mirroring",
+      "properties": {
+        "value": {
+          "description": "The items on the page",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/WorkloadNetworkPortMirroring"
+          }
+        },
+        "nextLink": {
+          "description": "URL to get the next page if any",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "WorkloadNetworkVMGroupProperties": {
+      "description": "NSX VM Group Properties",
+      "properties": {
+        "displayName": {
+          "description": "Display name of the VM group.",
+          "type": "string"
+        },
+        "members": {
+          "description": "Virtual machine members of this group.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "status": {
+          "description": "VM Group status.",
+          "type": "string",
+          "enum": [
+            "SUCCESS, FAILURE"
+          ],
+          "x-ms-enum": {
+            "name": "VMGroupStatusEnum",
+            "modelAsString": true
+          },
+          "readOnly": true
+        },
+        "revision": {
+          "description": "NSX revision number.",
+          "type": "integer",
+          "format": "int64"
+        }
+      }
+    },
+    "WorkloadNetworkVMGroup": {
+      "description": "NSX VM Group",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/WorkloadNetworkVMGroupProperties",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "WorkloadNetworkVMGroupsList": {
+      "description": "A list of NSX VM Groups",
+      "properties": {
+        "value": {
+          "description": "The items on the page",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/WorkloadNetworkVMGroup"
+          }
+        },
+        "nextLink": {
+          "description": "URL to get the next page if any",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "WorkloadNetworkVirtualMachineProperties": {
+      "description": "NSX Virtual Machine Properties",
+      "properties": {
+        "id": {
```

Comment:
id, name, and type are top level properties, not in the properties bag.  If you need an internal "type" of VM then please name it something different

Reply:
Fixed 👍
Removed id field, changed name field to displayName, and changed type field to vmType.